package.json

```diff
@@ -72,7 +72,6 @@
 	"pnpm": {
 		"overrides": {
 			"@types/react": "^17.0.71",
-			"react": "^17.0.2",
```

Comment:
I approved this original change, but frankly I think it shouldn't have been. it may have existed due to pnpm issues at the time, but we should not override dependencies like this in the monorepo without a very good reason.